app/controllers/comments_controller.rb

```diff
@@ -1,9 +1,9 @@
 class CommentsController < ApplicationController
+  before_action -> { authenticate_user! }, only: :create
```

Comment:
more pretty, I think
```suggestion
  before_action :authenticate_user!, only: :create
```